yb-voyager/src/queryparser/queryParser.go

```diff
@@ -0,0 +1,33 @@
+package queryparser
```

Comment:
I think it sort of makes sense to dump all the functions related to query parsing into a separate package. 

@priyanshi-yb consider moving the functions here if possible. 
Ideally we'd want to abstract away all the parsing to this package, and have our main packages (cmd, etc) not be aware of pg_query at all. We can discuss if that's worth doing or not..

Reply:
yes, was thinking about this as the functions I introduced in the analyze-schema are not isolated completely from that code. Will have to think of a proper way of separating them in this package, I will come up with something and then we can discuss it.

Reply:
Yeah lets discuss if/when we want to do that.

I explored shifting all that from analyzeSchema to this package, but it was a lot of work.